docs/testrunner-toolkit/configuration/common-syntax.md

```diff
@@ -176,6 +176,29 @@ __Example__:
 
 ⚠️ `registry` configuration is only supported in latest cypress docker image. No other frameworks, or Sauce Labs cloud.
 
+## `defaults`
+
+__Description__: Field for defining default settings.
+
+__Type__: *object*
+
+__Example__:
+```yaml
+defaults:
+  - mode: "sauce"
+```
+
+### `mode`
+
+__Description__: Specifies the default environment in which the tests should run. Choice: docker|sauce. If there is no setting for a suite, it should use this default setting.
```

Comment:
Also, I think the description should include the fact that this value is overridden when `mode` is set as a command line parameter.